apps/journeys-admin/src/components/SingleJourney/CardOverview/CardOverview.stories.tsx

```diff
@@ -11,12 +13,26 @@ const CardOverviewDemo = {
 }
 
 const Template: Story<CardOverviewProps> = ({ ...args }) => (
-  <CardOverview {...args} />
+  <MockedProvider>
+    <CardOverview {...args} />
```

Comment:
```suggestion
    <CardOverview {...args} slug={'my-journey'} />
```
In this case, the slug value is a valid value that we want to reuse for all stories (in previous tests, the value of some props were hard-coded to invalid values to fix linting)